README.md

```diff
@@ -17,3 +17,18 @@ We are also using [Tailwind](https://tailwindcss.com/) for the User Interface.
 For the Laravel backend to run, you will need a server (Apache or Nginx) and PHP. You can have these installed directly on your machine, or use a virtual environment. (see instructions on the [Laravel site](https://laravel.com/docs/7.x))
 
 Find more detailed instructions here: https://github.com/CodeHubOrg/codehub-mentorships/wiki/Getting-started
+
+## Authenticate
+
+Some routes in this application require authentication.  In order to access these routes you will need to ensure you have a database running locally that you can run the Laravel migrations against using this command (run from the project route):
+
```

Comment:
Might be good to add here that you need to run 'composer dump-autoload` first to load the UserTableSeeder class, if you are not setting the project up from scratch? - took me a bit to figure out!